lib/docblockr-worker.js

```diff
@@ -1062,13 +1065,16 @@ module.exports =
           }
         }
         for(i=0; len = out.length, i < len; i++) {
-          snippet+= '\n ' + this.prefix + (out[i] ? (this.indentSpaces + out[i]) : '');
+          snippet+= '\n' + this.prefix + (out[i] ? (this.indentSpaces + out[i]) : '');
         }
       }
       else
-        snippet+= '\n ' + this.prefix + this.indentSpaces + '${0:' + this.trailing_string + '}';
+        snippet+= '\n' + this.prefix + this.indentSpaces + '${0:' + this.trailing_string + '}';
+
+			if(this.parser.settings.commentType == "block") {
+				snippet+= '\n' + this.parser.settings.commentCloser;;
+			}
```

Comment:
The indention on thees lines seems broken. Also there is an extra semicolon at the end of the line.